weighter.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
+import os
+import ROOT as r
+import shlex, subprocess
+import argparse
+
+EOS_PATH = "root://eoslhcb.cern.ch/eos/ship/data/IS_experiment"
+FULL_MUONS_PATH = 'root://eoslhcb.cern.ch/eos/ship/data/Mbias/pythia8_Geant4-withCharm_onlyMuons_4magTarget.root'
+SLAVE_CMD = '''python2 code/slave.py ''' \
+            '''--params {params} ''' \
+            '''-f /shield/worker_files/sampling_is/''' \
+            '''muons.root ''' \
+            '''--results /output/result.json ''' \
+            '''--hists /output/hists_{tag}.root --seed {seed} ''' \
+            '''--xs_path {xs_path}'''
+
+def loss(x):
```

Comment:
How do you handle the two muon polarities here?

Reply:
Should I save muon polarities somewhere? or I can just look at the sign of x?

Reply:
The problem is that the formula for the loss depends on the muon polarity. Once it's calculated it is not needed anymore, but you need to be careful that it is taken into account for the calculation.

Reply:
Yes, I understand, but how I can extract the information about polarities?

Reply:
Never mind, if you save the the `x` in the `xs` array, it has already been accounted for. But in general, via the PID: Muons are 13 and anti-muons are -13.